CRM/Financial/BAO/FinancialItem.php

```diff
@@ -295,13 +295,28 @@ public static function checkContactPresent($contactIds, &$error) {
    * @return object CRM_Core_DAO
    */
   public static function getPreviousFinancialItem($entityId, $entityTable = 'civicrm_line_item') {
+    $liabilityAccountType = CRM_Core_OptionGroup::getValue('financial_account_type', 'Liability', 'name');
     $queryParams = array(
       1 => array($entityId, 'Integer'),
```

Comment:
throughout system entity_table always is entered in schema before entity_id. Stylistically would be nice to keep that order.